src/features/game/expansion/components/resources/Tree.tsx

```diff
@@ -256,14 +276,22 @@ export const Tree: React.FC<Props> = ({ treeIndex, expansionIndex }) => {
       />
 
       {chopped && (
-        <>
+        <div
+          style={{
+            position: "absolute",
+            width: `${GRID_WIDTH_PX * 2}px`,
+            height: `${GRID_WIDTH_PX * 2}px`,
+            bottom: 0,
+          }}
+        >
```

Comment:
This creates a 2x2 grid for the stump to sit in. Compared to the 2x3 grid the tree sits in.

![stump](https://user-images.githubusercontent.com/41215134/184062044-1db0d021-3572-46e3-93ce-a58e23d6b8c3.png)